app/src/main/java/xyz/whereuat/whereuat/gcm/WuaGcmListenerService.java

```diff
@@ -74,16 +73,16 @@ public void onMessageReceived(String from, Bundle data) {
      */
     private void handleAtRequest(Bundle data) {
         final String from_phone = data.getString(Constants.GCM_FROM_KEY);
-        // Build and execute a query to determine if the contact is autoshared.
-        QueryCommand query = ContactUtils.buildSelectContactByPhoneCommand(this, from_phone,
+        final QueryCommand query = ContactUtils.buildSelectContactByPhoneCommand(this, from_phone,
```

Comment:
why not define this in the `Runnable`? Closer to where it's used is better (Copywrite Rouslan Soloumakin, Google Chrome Spellchecker Dev extraordinaire)